cluster/gce/configure.sh

```diff
@@ -48,12 +45,12 @@ PKG_PREFIX=${PKG_PREFIX:-"cri-containerd-cni"}
 VERSION_METADATA="version"
 VERSION=$(fetch_metadata "${VERSION_METADATA}")
 if [ -z "${VERSION}" ]; then
-  VERSION=$(curl -f --ipv4 --retry 6 --retry-delay 3 --silent --show-error \
-    https://storage.googleapis.com/${DEPLOY_PATH}/latest)
+  echo "Version is not set."
```

Comment:
need to edit comment lines 43-44 above...

Reply:
Will do. Thanks!

Reply:
Done.